### DISCUSSION TITLE
Link A Link B Link C removal of confusing and duplicate features in joomla 4

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#32179] [General]
Link A Link B Link C removal of confusing and duplicate features in joomla 4

It also makes joomla 4 a simpler and easier to use.

joomla 4 is without release date and delayed. We need to ship joomla 4 as soon as we can, why you want to introduce confusing and duplicate features in joomla 4 ?

If you use joomla 3 with Link A Link B Link C you can upgrade later if there will be need using extension or in j 4.1 or j 5

Why not to do this?

Link A Link B Link C should be removed, its confusing and duplicate feature in joomla 4

Theus who need Link A Link B Link C can install it and use it as an own package as Weblinks users do

Keeping and maintaining confusing and duplicate features in joomla 4 is bad design

**Comment by brianteeman** [comment]
TROLL!!!!

**Reply by ghost** [reply]
Brian in my opinion introducing confusing and duplicate features in joomla 4 is bad design

**Reply by brianteeman** [reply]
And you obviously have never used joomla before because this functionality is NOT new and has been present since  **20 Dec 2011**

**Reply by ghost** [reply]
Brian MooTools was in joomla way back 20 Dec 2011 according to your logic we should leave MooTools in joomla 4

Link A Link B Link C is confusing and duplicate features so leaving it in joomla 4 is simply bad design, or do you think its good to ship joomla 4 with confusing and duplicate features ?

**Reply by brianteeman** [reply]
removing mootools doesnt remove data

**Reply by ghost** [reply]
Brian simple question to you do you think its good to ship joomla 4 with confusing and duplicate features ?

**Reply by ghost** [reply]
removing weblinks does remove data and that was done in joomla

**Reply by brianteeman** [reply]
No it did not. Weblinks was not removed from existing sites

**Reply by ghost** [reply]
but how can new sites use weblinks ?

**Reply by drmenzelit** [reply]
You can install it. It is an own package.

**Reply by ghost** [reply]
menzelit thats why Link A Link B Link C should be removed, its confusing and duplicate feature in joomla 4

Theus who need Link A Link B Link C could use it is an own package as Weblinks users do

**Reply by drmenzelit** [reply]
You cannot compare weblinks with Link A, Link B and Link C in articles... I don't understand your argumentation.

**Reply by ghost** [reply]
menzelit why you cannot compare weblinks with Link A, Link B and Link C ?

**Comment by ghost** [comment]
Here you can see why Link A Link B Link C removal in joomla 4 should be done https://github.com/joomla/joomla-cms/issues/30280

**Comment by ghost** [comment]
Theus in j4 who need Link A Link B Link C could use it is an own package as Weblinks users do

**Reply by brianteeman** [reply]
Good luck in achieving that magic

**Reply by ghost** [reply]
first we need pr to remove Link A Link B Link C from j4 core

**Reply by brianteeman** [reply]
For the hundredth time. If you remove those links then every site that is using them already will be broken when they upgrade to joomla 4. Unless you can write an update script that will detect if the fields are being used, create the required custom fields and then copy the data from the links to the custom fields you cannot remove the fields. As for the links being a separate package like weblinks then I suggest that you actually spend more than a second to think about it and you will realise that there is no comparison to weblinks.

**Reply by ghost** [reply]
So as we can see, the comment to keep those links is pure nonsense. Brian if you use these links you don't need to upgrade to  j4 you can stay with j3 . No need to drag these links to joomla 4 and make joomla 4 confusing and with duplicate features.

Brian better tell us how you use these links in j4 , we want to see why you for the hundredth time claim these links are not confusing and with duplicate features in j4?
